moler/cmd/at/set_functionality_level.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+"""
+AT+CFUN
+
+AT commands specification:
+google for: 3gpp specification 27.007
+(always check against latest version of standard)
+"""
+
+__author__ = 'Kamil Pielka'
+__copyright__ = 'Copyright (C) 2021, Nokia'
+__email__ = 'kamil.pielka@nokia.com'
+
+from moler.cmd.at.genericat import GenericAtCommand
+
+
+class SetFunctionalityLevel(GenericAtCommand):
+    """
+    Command to set the level of functionality for UE. Example output:
+
+    AT+CFUN=1
+    OK
+    """
+    def __init__(self, fun_level, rst=None, connection=None, prompt=None, newline_chars=None, runner=None):
+        """Create instance of SetFunctionalityLevel class
+        :param fun_level: UE's level of functionality
+                          Possible valueATs are:
+                          0 - minimum functionality
+                          1 - full functionality
+                          2 - disable phone transmit RF circuits only
+                          3 - disable phone receive RF circuits only
+                          4 - disable phone both transmit and receive RF circuits
+                          5...127 - reserved for manufacturers as intermediate states between full and minimum
+                                    functionality
+        :param rst: whether to restart UE before setting level of functionality
+                    None - there is not provide openly any value in command (AT command uses default value)
+                    0 - not reset UE before setting level of functionality (default value)
+                    1 - reset UE before setting level of functionality (level of functionality must be 1)
+        :param connection: moler connection to device
+        :param prompt: start prompt (on system where command cd starts)
+        :param newline_chars: Characters to split lines
+        :param runner: Runner to run command
+        """
+        super(SetFunctionalityLevel, self).__init__(connection, operation='execute', prompt=prompt,
```

Comment:
Please use one method of passing arguments (connection=connection).

Reply:
Corrected